README.md

```diff
@@ -7,3 +7,11 @@ This is a package of utility functions and test fixtures for suitcase packages.
 ```
 pip install suitcase-utils
 ```
+
+## Quick Start
+
+TODO
+
+## Documentation
+
+See the [suitcase documentation](https://nsls-ii.github.io/suitcase).
```

Comment:
As Maksim pointed out on the suitcase-csv version, it would be better to use https://blueskyproject.io/suitcase/

Reply:
thanks